src/Control/Controller.php

```diff
@@ -61,6 +63,8 @@ class Controller extends RequestHandler implements TemplateGlobalProvider
     protected $templates = [];
 
     /**
+     * @deprecated 4.1.0...5.0.0 Add this controller's url to
+     * SilverStripe\Security\BasicAuthMiddleware.ExcludeURLPatterns property instead of setting false
```

Comment:
old property still suggested `urlPatterns` is the new one right?

Reply:
oops sorry.